pkg/devfile/adapters/common/command_test.go

```diff
@@ -486,52 +339,60 @@ func TestGetBuildCommand(t *testing.T) {
 	command := "ls -la"
 	component := "alias1"
 	workDir := "/"
-	validCommandType := common.DevfileCommandTypeExec
 	emptyString := ""
 
 	var emptyCommand common.DevfileCommand
 
 	tests := []struct {
-		name           string
-		commandName    string
-		commandActions []common.DevfileCommandAction
-		wantErr        bool
+		name         string
+		commandName  string
+		execCommands []common.Exec
+		wantErr      bool
 	}{
 		{
 			name:        "Case: Default Build Command",
 			commandName: emptyString,
-			commandActions: []common.DevfileCommandAction{
+			execCommands: []common.Exec{
 				{
-					Command:   &command,
-					Component: &component,
-					Workdir:   &workDir,
-					Type:      &validCommandType,
+					CommandLine: command,
+					Component:   component,
+					WorkingDir:  workDir,
+					Group:       &versionsCommon.Group{Kind: buildGroup, IsDefault: true},
 				},
 			},
 			wantErr: false,
 		},
 		{
-			name:        "Case: Custom Build Command",
-			commandName: "customcommand",
-			commandActions: []common.DevfileCommandAction{
+			name:        "Case: Build Command passed through the odo flag",
```

Comment:
Although the previous tests has incorrect names, can you update the name of the cases in this PR? Ex. Each should be numbered Case 1, Case 2, etc in the tests.